docs/source/package_orderers.rst

```diff
@@ -0,0 +1,86 @@
+================
+Package Orderers
+================
+
+Rez's default version resolution algorithm will always sort by the latest alphanumeric
+version. However, package orderers allow you to customize this functionality globally,
+or at a per package level.
+
+Configuring package orderers
+============================
+
+Package ordering settings are not configured in each package's ``package.py``. Instead they are configured in the
+``rezconfig.py`` via the :data:`package_orderers` setting.
+
+Sorted Order
+------------
+
+This is the default orderer the sorts based on the package's :attr:`version` attribute.
+
+You can optionally explicitly request this orderer like this:
+
+.. code-block:: python
+
+    package_orderers = [
+        {
+            "type": "sorted", # Required
+            "descending": True, # Required
+            "packages": ["python"] # Optional, if not supplied, orderer applies to all packages
+        }
+    ]
+
+Per Family Order
+----------------
+
+This orderer allows you to define different orderers to different package families.
```

Comment:
We should document that `orderers` can be any supported orderer.

Reply:
@JeanChristopheMorinPerso Can you explain a bit more what you mean?